packages/wonder-blocks-data/src/index.js

```diff
@@ -21,8 +22,21 @@ export type {
 export const initializeCache = (source: ResponseCache): void =>
     ResCache.Default.initialize(source);
 
-export const fulfillAllDataRequests = (): Promise<ResponseCache> =>
-    RequestTracker.Default.fulfillTrackedRequests();
+export const fulfillAllDataRequests = (): Promise<ResponseCache> => {
+    if (!Server.isServerSide()) {
+        return Promise.reject(
+            new Error("Data requests are not tracked when client-side"),
+        );
+    }
```

Comment:
This would've helped catch that we weren't setting WB to server side mode in our route tests if we had it place before my 2 hour search for that bug the other day.